src/components/pages/Home/HomePage.jsx

```diff
@@ -1,27 +1,44 @@
 import React from 'react';
 import PageLayout from '<templates>/PageLayout/PageLayout';
-import Text from '<atoms>/Text/Text';
 import headerMapper from '<helpers>/headerMapper';
 import displayLoader from '<helpers>/displayLoader';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Image from '<atoms>/Image/Image';
+import ImageUrl from '<image>/homepage.png';
 import items from './navItems';
+import Title from '<atoms>/Title/Title';
+import SearchBar from '<molecules>/SearchBar/SearchBar';
 
 const HomePage = () => {
   const navItems = headerMapper(items);
 
   return (
     <PageLayout navItems={navItems} callback={displayLoader}>
-      <Text
-        content="true"
-        textTransform="uppercase"
-        padding="md"
-        fontWeight="normal"
-        display="inline"
-        fontSize="normal"
-        textAlign="center"
-        color="primary"
-      >
-        Welcome to Authors Haven
-      </Text>
+      <FlexContainer justifyContent='space-between'>
+        <FlexContainer flexDirection='row'>
+          <FlexContainer
```

Comment:
i think you should use the already existing image component since it is being reused.